4-nodarbiba/uzdevumi/TomsStals/objects/js/app.js

```diff
@@ -33,4 +33,41 @@
 // 8. Izvadīt konsolē CV objekta īpašības languages objekta eng īpašības name un level apvienojumu.
 //    -- piem., vardsUzvards.languages.lv objekta īpašību apvienojums izskatītos šādi:
 //       Latviešu - Native 
-//       Un tiktu panākta ar string literal pierakstu `${vardsUzvards.languages.lv.name} - ${vardsUzvards.languages.lv.level}`
\ No newline at end of file
+//       Un tiktu panākta ar string literal pierakstu `${vardsUzvards.languages.lv.name} - ${vardsUzvards.languages.lv.level}`
+
+
+const cv = {
+
+    firstName: "Jeff",
+    lastName:  "Bezos",
+    email: "jeff@amazon.com",
+    telNr: +1-541-754-3010,
+    workExperience: [
+        "SIA Amazon",
+        "SIA Blue Origin",
+        "SIA Bezos Unlimited",
+        "SIA Jeffy Home Deliveries"
+    ],
+
+    languages: {
+        lv:{
+            name:"Latviesu",
+            level: "Native"
+        },
+        eng:{
+            name:"English",
+            level:"Professional"
+        },
+        dk:{
+            name:"Danish",
+            level:"Basic",
+        }
+    }
+}
+
+//Task 6.
+console.log(cv.firstName + " " + cv.lastName)
```

Comment:
Šis, protams, strādā un ir ok.
Bet labi būtu izmantot "string literals" sintaksi:

`'${cv.firstName} ${cv.lastName}'`